### ISSUE TITLE
Update `/presentations`

### ISSUE THREAD

**Issue by martindale** [tendermint/tendermint.com#137]
The [list of presentations](https://tendermint.com/presentations) looks like it needs some love. :sparkling_heart:

**Comment by martindale** [comment]
Candidate for inclusion: ["Many Chains, One Ecosystem", by Sunny Agarwal](https://www.youtube.com/watch?v=XN_SAX73AMg)

**Comment by martindale** [comment]
Candidate for inclusion: ["Many Chains, Many Tokens, One Ecosystem", by Brian Crain](https://www.youtube.com/watch?v=FJrio52MUsk)

**Comment by nylira** [comment]
Agreed. I'll ask the comms team for a list of all our of presentations. The Many Chains talk is already on the Cosmos website though. https://cosmos.network/developers/academy
